roles/ci_dcn_site/tasks/scaledown_site.yaml

```diff
@@ -0,0 +1,238 @@
+
+- name: Delete rabbitmqcluster
+  vars:
+    az_to_cell_map:
+      az0: cell1
+      az1: cell2
+      az2: cell3
+  ansible.builtin.shell: |
+    oc delete rabbitmqclusters rabbitmq-{{ az_to_cell_map[_az_to_scaledown] }}
+
+- name: Delete the cinder-volume service
+  kubernetes.core.k8s_exec:
+    api_key: "{{ _auth_results.openshift_auth.api_key }}"
+    namespace: openstack
+    pod: cinder-scheduler-0
+    command: >-
+      "cinder-manage service remove cinder-volume cinder-volume-{{ _az_to_scaledown }}-0@ceph"
+
+- name: Fetch ceph-conf-files secret
+  kubernetes.core.k8s_info:
+    api_key: "{{ _auth_results.openshift_auth.api_key }}"
+    kind: Secret
+    name: ceph-conf-files
+    namespace: openstack
+  register: secret_info
+
+- name: Save secret data to files
+  copy:
+    content: "{{ secret_info.resources[0].data[key] | b64decode }}"
+    dest: "/tmp/{{ key }}"
+  loop: "{{ secret_info.resources[0].data.keys() }}"
+  loop_control:
+    loop_var: key
+
+- name: Delete the Ceph cluster's secrets of removed cluster and default site cluster
+  kubernetes.core.k8s:
+    api_key: "{{ _auth_results.openshift_auth.api_key }}"
+    kind: Secret
+    name: "/tmp/{{ item }}"
+    namespace: openstack
+    state: absent
+  loop:
+    - "ceph-conf-files-{{ _az_to_scaledown }}"
+    - "ceph-conf-files"
+
+- name: Find all ceph variable files
+  register: all_ceph_conf_files
+  ansible.builtin.find:
+    paths: "./"
+    patterns: "az*.c*"
+    recurse: false
+
+- name: Set fact for base64-encoded file data of ceph-conf-files Secret
+  set_fact:
+    file_data: "{{ file_data | default({}) | combine({ item | basename: (lookup('file', item) | b64encode) }) }}"
+  loop: "{{ all_ceph_conf_files.files | map(attribute='path') | reject('search', _az_to_scaledown) | list }}"
+
+- name: Recreate the secret while omitting deleted ceph cluster
+  kubernetes.core.k8s:
+    api_key: "{{ _auth_results.openshift_auth.api_key }}"
+    state: present
+    definition:
+      kind: Secret
+      metadata:
+        name: ceph-conf-files
+        namespace: openstack
+      type: Opaque
+      data: "{{ file_data }}"
+
+- name: Get compute nodes from the scale-downed AZ
+  register: az_compute_hosts
+  kubernetes.core.k8s_exec:
+    api_key: "{{ _auth_results.openshift_auth.api_key }}"
+    namespace: openstack
+    pod: openstackclient
+    command: >-
+      sh -c "openstack compute service list -c ID -c Zone -f value | grep {{ _az_to_scaledown }} | awk '{print $1}'"
+
+- name: Disable the compute service on scale-downed compute nodes
+  loop: "{{ az_compute_hosts }}"
+  kubernetes.core.k8s_exec:
+    api_key: "{{ _auth_results.openshift_auth.api_key }}"
+    namespace: openstack
+    pod: openstackclient
+    command: >-
+      openstack compute service set {{ item }} nova-compute --disable
+
+- name: Stop the ovn and nova-compute containers and remove the systemd unit files
```

Comment:
I see that steps like this are implementing the documentation which is exactly what we want. This way when the ci-framwork runs these tasks we can quickly know that what's in the documentation is correct (assuming it passes). Thanks.

https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0/html/maintaining_the_red_hat_openstack_services_on_openshift_deployment/assembly_scaling-data-plane-nodes#proc_removing-an-OpenStackDataPlaneNodeSet-resource_scaling